.github/workflows/non-cot-pr-build-and-e2e-tests.yml

```diff
@@ -66,8 +65,8 @@ jobs:
                   if-no-files-found: ignore
                   retention-days: 5
 
-    cot-api-tests-run:
-        name: Runs API tests with COT enabled.
+    non-cot-api-tests-run:
+        name: Runs API tests with COT disabled.
```

Comment:
Resolving this conversation now, since I can already see the job name as `non-hpos-api-tests-run` with name `Runs API tests with HPOS disabled.`
<img width="405" alt="Screenshot 2023-10-11 at 2 32 37 PM" src="https://github.com/woocommerce/woocommerce/assets/4509348/56641a74-e672-4863-8570-9cec8f913e97">